bricks/permission_handler/__brick__/{{~ _pubspec_dependency.yaml }}

```diff
@@ -0,0 +1 @@
+  permission_handler: ^8.3.0
```

Comment:
same here

Reply:
@doannimble explained above ☝️, having a new line will inject 2 lines into the target file `bricks/template/__brick__/{{project_name.snakeCase()}}/pubspec.yaml`